net/ser2net/files/etc/init.d/ser2net

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh /etc/rc.common
+START=75
+STOP=10
+CONFIGFILE="/tmp/ser2net.conf"
+
+start() {
+	pidof ser2net | awk '{print $2}' > /tmp/ser2net
+	PID=$(cat /tmp/ser2net)
+	rm /tmp/ser2net
+    if [ ! $PID ]; then
+		update_configfile
+        ser2net -c "$CONFIGFILE"
+
+    else
+        echo "Ser2Net: is already running"
+    fi
+}
+
+stop() {
+	pidof ser2net | sed -e "s/$$//g" > /tmp/ser2net_pid
+	PID=`cat /tmp/ser2net_pid`
+	rm -f /tmp/se2net_pid
+	kill -9 $PID 	
+}
+
+update_configfile() {
+	echo "Ser2Net: Generating "$CONFIGFILE" based on /etc/config/ser2net (in cbi format)."
+	echo "#This file is generated based on /etc/config/ser2net." > "$CONFIGFILE"
+	i=0
+	while [ true ]
+	do
+	        tcpport=`uci get ser2net.@proxy["$i"].tcpport 2> /dev/null`
+	        if [ -z $tcpport ]; then
+	                break
+	        fi
+	        state=`uci get ser2net.@proxy["$i"].state 2> /dev/null`
+	        timeout=`uci get ser2net.@proxy["$i"].timeout 2> /dev/null`
+	        device=`uci get ser2net.@proxy["$i"].device 2> /dev/null`
+	        options=`uci get ser2net.@proxy["$i"].options 2> /dev/null`
+
+	        if [ -z "$options" ]; then
+	                echo "$tcpport":"$state":"$timeout":"$device" >> "$CONFIGFILE"
+	        else
+	                echo "$tcpport":"$state":"$timeout":"$device":"$options" >> "$CONFIGFILE"
```

Comment:
you know this can be more than just a port right? you can put in things like ```ipv4,tcp,localhost,2000``` in that first field even.

Reply:
If I check https://linux.die.net/man/8/ser2net and look at the Configuration section they also mention it as TCP port.

Their setup is: TCP port : state : timeout : device : options

Reply:
That man file is out of date.  Look at the docs within the existing /etc/ser2net.conf

Reply:
Ok, so the new name is "network port". So changing it to "networkport" will be sufficient?
[ser2net.conf.txt](https://github.com/openwrt/packages/files/1576422/ser2net.conf.txt)